config/main.py

```diff
@@ -1227,6 +1227,92 @@ def validate_patch(patch):
     except Exception as e:
         raise GenericConfigUpdaterError(f"Validate json patch: {patch} failed due to:{e}")
 
+
+def multiasic_validate_single_file(filename):
+    ns_list = [DEFAULT_NAMESPACE, *multi_asic.get_namespace_list()]
+    file_input = read_json_file(filename)
+    file_ns_list = [DEFAULT_NAMESPACE if key == HOST_NAMESPACE else key for key in file_input]
+    if set(ns_list) != set(file_ns_list):
+        diff = set(ns_list) - set(file_ns_list)
+        click.echo("Input file {} must contain all asics config. Missing {}".format(filename, diff))
+        raise click.Abort()
+
+
+def load_sysinfo_if_missing(asic_config):
+    platform = asic_config.get("DEVICE_METADATA", {}).\
```

Comment:
>asic_config.get("DEVICE_METADATA", {}) [](http://example.com/codeflow?start=15&length=38)

You can create a var just for DEVICE_METADATA value, and prevent breaking long line, which is not easy to read.